_blog.yml

```
@@ -6292,3 +6292,15 @@
     - robotics
     - community
     - open-source
+
+- local: hf-mcp-remote
```

```suggestion
- local: building-hf-mcp
```
maybe more catchier like this WDYT?